src/server/svr_func.c

```diff
@@ -5236,7 +5239,24 @@ is_vnode_prov_done(char * vnode)
 			id, prov_vnode_info->pvnfo_vnode))
 		return;
 	}
+	
+	/* check if IS_HOOK_CHECKSUMS is processed if not then
+ 	 * come back after 5 sec and check again, this affirms
+ 	 * the hook files are in sync before starting the job */
 
+	if (!((mom_svrinfo_t *)(pnode->nd_moms[0]->mi_data))->msr_is_hook_checksums_done
+			&& !sync_mom_hookfiles_proc_running && !do_sync_mom_hookfiles){
```

Comment:
Hi Dilip,
Thanks for the review comment.
But how can we say all the nodes assigned to the provisioning job will reboot at the same time?
And this has to happen only when node up again.